mandible/metadata_mapper/directive/mapped.py

```diff
@@ -15,16 +16,17 @@ class Mapped(TemplateDirective):
 
     source: str
     key: Key
+    key_options: dict = field(default_factory=dict)
 
     def __post_init__(self):
         if self.source not in self.sources:
-            raise MetadataMapperError(f"source '{self.source}' does not exist")
+            raise MetadataMapperError(f"source {self.source!r} does not exist")
```

Comment:
```suggestion
            raise MetadataMapperError(f"source {repr(self.source)} does not exist")
```
Reading https://peps.python.org/pep-0498/#s-r-and-a-are-redundant makes it seem `repr()` is useable here and I think it's clearer.

Reply:
Interesting, sure I can see that.